### ISSUE TITLE
Remove /lib

### ISSUE THREAD

**Issue by J-N-K** [openhab/openhab-addons#8682]
This is to keep track of binary data in the repo.

| Binding | PR | 
| ---|---|
| bluetooth.ruuvitag |#8701 |
| cbus |#8684|
| plclogo |#8687|
| rfxcom |#8700|

To prevent adding new libs, the embedding code should be removed from the pom.

**Comment by kaikreuzer** [comment]
> To prevent adding new libs, the embedding code should be removed from the pom.

We should probably also [add some documentation](https://www.openhab.org/docs/developer/guidelines.html#d-language-levels-and-libraries) as we currently seem to be lacking any information on how to deal with required external libraries that are not "default". I myself regularly have issue remembering how we expect it to be done...

**Comment by J-N-K** [comment]
I‘m pretty sure I wrote something about that. Can‘t remember where though.

**Comment by kaikreuzer** [comment]
Haha, if you find out, I'd be interested in it! 😎

**Comment by J-N-K** [comment]
https://github.com/openhab/openhab-docs/pull/1199

Maybe we should add that bundles that are not available from a remote repo can be added to our openhab-addons-dep bintray

**Comment by kaikreuzer** [comment]
> openhab/openhab-docs#1199

Ah, thanks!

> Maybe we should add that bundles that are not available from a remote repo can be added to our openhab-addons-dep bintray

Would be a nice addition. Want to create a PR for it? Especially on how to get it added to openhab-addons-deps.

**Comment by martinvw** [comment]
Hello @kaikreuzer or @wborn, could it be that this dependency was not migrated, could you quickly outline what I can do do to help this to be fixed? Thanks a lot!

**Comment by wborn** [comment]
Thanks for reporting this. Maybe @kaikreuzer can add them to Artifactory? Jenkins doesn't run into it because it only removes `org.openhab` artifacts from the local Maven repo to improve build times.

**Comment by kaikreuzer** [comment]
I removed bintray from the artifactory releases repo yesterday and am currently looking at adding missing stuff to Artifactory directly.
Reference build is https://ci.openhab.org/view/Sandbox/job/sandbox-openhab3-release/ - once this is green, it should be fine.
Will finish it by the end of this month.

**Comment by martinvw** [comment]
If it helps you. Feel free to create a user for me, I can make some time today to upload the missing artifacts there. Otherwise could luck and let me know if there is something which I could do to help

**Comment by wborn** [comment]
The [distro build](https://ci.openhab.org/job/openHAB3-Distribution/2351/console) also seems to have some issue downloading artifacts:

`[ERROR] Failed to execute goal org.apache.karaf.tooling:karaf-maven-plugin:4.3.1:kar (default-kar) on project openhab-addons: Failed to create archive: Could not find artifact org.openhab:com.caucho.quercus:jar:4.0.45 in openhab-release (https://openhab.jfrog.io/openhab/libs-release)`

**Comment by martinvw** [comment]
FTR, this is what IntelliJ reports to be missing

```
Cannot resolve fi.tkgwf.ruuvi:ruuvitag-common:1.0.0

Cannot resolve com.github.tavalin.orvibo-sdk:s20-sdk:0.0.6-nmu1

Cannot resolve com.github.pathec:zway-lib:0.2.9.OH

Cannot resolve com.github.jpharvey:cgateinterface:1.1.0-JH-dev

Cannot resolve org.tellstick:javatellstick:1.1

Cannot resolve com.leapmotion.leap:leap-java:2.0.0

**Comment by kaikreuzer** [comment]
Many thanks @martinvw! I have uploaded all those libs to Artifactory now.
Distro build is also green now.

**Comment by kaikreuzer** [comment]
Sandbox build is green, so it all looks good now!

**Comment by wborn** [comment]
I noticed that both 2.5.x and 3.0.x use `https://openhab.jfrog.io/openhab/libs-release` as online repo @kaikreuzer. But it now only contains 3.0.2 addons. Which explains why everyone not running that version has issues installing them (even though bintray hasn't sunset yet):

* https://community.openhab.org/t/oh3-bindings-fail-to-downoad/121512/
* https://community.openhab.org/t/docker-2-5-12-error-core-karaf-internal-featureinstaller-failed-installing/121517

**Comment by kaikreuzer** [comment]
Hm, I wasn't aware that 2.5.12 also pointed to Artifactory... In some way it is good as it gives us the chance to make it working even after May 1. I have created a new local repo https://openhab.jfrog.io/artifactory/libs-online-2.5.12/ with all 2.5.12 artifacts and can include it into the virtual libs-release repo. Unfortunately, the 2.5.12 still failed finding them - possibly because it is looking for a version range and the metadata is missing.

Well, I have added the bintray location to the virtual repo again, so that people can still use everything as before for 3 more days. I'll then shift the further testing to after that date, since then it will anyhow not work anymore...
